server/target/pagination.go

```diff
@@ -0,0 +1,50 @@
+package target
```

Comment:
nit: our current convention is to have one Go file per package, so that if you see an identifier without a package prefix, you can tell that it is located in the file you're currently looking at

Reply:
Moved to target.go